abb_driver/rapid/ROS_festo_init.mod

```diff
@@ -0,0 +1,22 @@
+MODULE ROS_festo_init
+    
+LOCAL VAR bool has_been_init := FALSE;
+
+PROC main()
+    
+    IF (NOT has_been_init) THEN
+        ! Initialize Festo
+        Festo_init;
+    
+        ! Home Festo
+        Festo_homing;
+        
+        has_been_init := TRUE;
+        
+        WaitTime 5;

```

Comment:
why wait here?

Reply:
The wait is for the robot to home itself -- homing takes a while -- the rail moves slowly from wherever it is to one side, where it marks as 0 the place where there's a push against it / current spike.  (For now -- we're adding limit switches to make this more accurate).